website/.gitignore

```diff
@@ -27,6 +27,7 @@ yarn-error.log*
 
 # local env files
 .env*.local
+.env
```

Comment:
Can you remove this? I intentionally left it out so that someone could do `npx prisma db push` and update the dev database without having to configure anything.